src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/project.json

```diff
@@ -21,31 +21,36 @@
   },
 
   "dependencies": {
-    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.1, 4.0)"
+    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.5, 4.0)"
   },
 
   "frameworks": {
     "net45": {
+      "buildOptions": {
+        "define": [ "NET45" ],
+        "debugType": "portable"
+      },
       "dependencies": {
       }
     },
     "netstandard1.5": {
+      "buildOptions": {
+        "define": [ "NETSTANDARD11", "PORTABLE" ],
```

Comment:
First we need to fix AutoRest not to generate PORTABLE definition in code. Then remove it
e.g. https://github.com/Azure/azure-sdk-for-net/blob/7c79d8a0930c93917b3c1029ec2936423719ae15/src/ClientRuntime/Microsoft.Rest.ClientRuntime/RestException.cs#L12